cmd/cluster-operator-controller-manager/app/controllermanager.go

```diff
@@ -385,20 +390,23 @@ func GetAvailableResources(clientBuilder controller.ClientBuilder) (map[schema.G
 // controllers such as the clientBuilder.
 func CreateControllerContext(s *options.CMServer, clientBuilder controller.ClientBuilder, stop <-chan struct{}) (ControllerContext, error) {
 	versionedClient := clientBuilder.ClientOrDie("shared-informers")
+	kubeClient := clientBuilder.KubeClientOrDie("shared-informers")
 	sharedInformers := clusteroperatorinformers.NewSharedInformerFactory(versionedClient, ResyncPeriod(s)())
+	kubeSharedInformers := kubeinformers.NewSharedInformerFactory(kubeClient, 10*time.Minute)
```

Comment:
Could we use the ResyncPeriod function above or is the static ten minutes safe here?

Reply:
I would think that the static is ok, but will check

Reply:
The `ResyncPeriod` does some jittering to reduce the likelihood that each controller will be hitting the API server in lock-step with each other. We should use `ResyncPeriod` for the kube informers, too.